protocol/messenger_mailserver.go

```diff
@@ -1019,6 +1019,14 @@ func (m *Messenger) ToggleUseMailservers(value bool) error {
 	return nil
 }
 
+func (m *Messenger) TogglePeerSyncing(value bool) error {
```

Comment:
maybe best to put it in `protocol/messenger_settings.go`
and use a `request` object

similarly to:

```
func (m *Messenger) SetLightClient(request *requests.SetLightClient) error {
	return nodecfg.SetLightClient(m.database, request.Enabled)
}
```